demo/cypress/integration/axe.spec.js

```diff
@@ -0,0 +1,13 @@
+context('Accessibility', () => {
+  it('all component previews pass a basic accessibility check', () => {
+    cy.visit('/rails/view_components');
+    cy.get('li a').then((tags) => {
+      Array.from(tags)
+        .map((t) => t.href)
+        .forEach((href) => {
+          cy.visit(href);
+          cy.runAxe();
+        });
+    });
+  });
```

Comment:
This test essentially allows us to replace pa11y with an automated check within the cypress tests using `cypress-axe`. This list is against every preview example so we don't need to keep any lists in sync.

We can also run additional checks during scenarios if we want, e.g. to check again when targeted content is open. For now I've left it to this smoke check.